branch-diff.js

```diff
@@ -111,6 +111,10 @@ function printCommits (list, format) {
   } else {
     list = list.map((commit) => commitToOutput(commit, format === 'simple', ghId))
   }
+  
+  if (reverse) {
+    list = list.reverse();
+  }
```

Comment:
Reversed for `sha` would probably be surprising behaviour, the assumption you're making is that it'll only be used for `xargs` but I can imagine a few other use-cases for `sha` output. I vote for needing to explicitly supply `--reverse`.